hpcpy/client.py

```diff
@@ -331,16 +331,20 @@ def submit(
         # Add queue
         if queue:
             directives.append(f"-q {queue}")
+            context["queue"] = queue
 
         # Add walltime
         if walltime:
             _walltime = str(walltime)
             directives.append(f"-l walltime={_walltime}")
+            context["walltime"] = _walltime
 
         # Add storage
         if storage:
             storage_str = "+".join(storage)
             directives.append(f"-l storage={storage_str}")
+            context["storage"] = storage
+            context["storage_str"] = storage_str
```

Comment:
Both `storage` and `storage_str` are provided for interpolation, allowing the user the flexibility to iterate through the storage array or use the joined string.